doc/classes/RichTextLabel.xml

```diff
@@ -163,6 +170,19 @@
 				Installs a custom effect. [code]effect[/code] should be a valid [RichTextEffect].
 			</description>
 		</method>
+		<method name="is_menu_visible" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns whether the menu is visible. Use this instead of [code]get_menu().visible[/code] to improve performance (so the creation of the menu is avoided).
+			</description>
+		</method>
+		<method name="menu_option">
+			<return type="void" />
+			<argument index="0" name="option" type="int" />
+			<description>
+				Triggers a right-click menu action by the specified index. See [enum MenuItems] for a list of available indexes.
+			</description>
+		</method>
```

Comment:
I'm not sure if there's any use in exposing this to the GDScript, probably should be hidden (start with `_`).

Reply:
done